README.yaml

```diff
@@ -28,9 +28,21 @@ include:
   - "docs/targets.md"
 
 usage: |-
-  1. Run `make init`
-  2. Run `COMPONENTS_BUILD=true/false make run` to start local server
-  3. Run `make build` to generate static site in `public/`
+  1. Download [Docker](https://www.docker.com/).
+  2. Install the [Atom editor](https://atom.io/)(optional).
+
+    *After installation, run `make deps/atom`.
+
+    *Select _Activate the Atom_ --> _Install Shell Commands_ menu item.
+
+    *Ensure that /usr/local/bin is in the PATH.
+  3. Run the following commands for faster development:
```

Comment:
3. This is some text
    ```
    this is a code block
    /* with some code */
   ````
4. This is the next bullet

@drnickiw click edit on this comment to see how to write the markdown. Basically you need to use spaces (not tabs) to indent the code block. Then it should work.